server/bin/pbench-remote-sync-package-tarballs

```diff
@@ -20,11 +20,16 @@ case $# in
         :
         ;;
     *)
-        echo "Usage: $PROG <satellite-config> <output-tar-ball>"
+        echo "Usage: $PROG <satellite-config> <output-tar-ball>" >&2
         exit 1
         ;;
 esac
 
+if [[ -z "${dir}" || -z "${PROG}" ]]; then
```

Comment:
Shouldn't this be checked after line 17 above since `$PROG` is referenced in the above `case` construct?